### DISCUSSION TITLE
Lost GIT tab

### DISCUSSION THREAD

**Discussion by shayavh** [IBAHCM/RPiR#317] [Archive] [answered]
Hi @richardreeve @soniamitchell, 
I don't know how, but I noticed I lost my GIT tab in my project. 
How do I solve this?
Thanks!

**Comment by richardreeve** [comment, marked as answer]
Eek. Have you just updated your operating system or anything like that? If so it may be that git is no longer installed on your computer...

In any event, I would try (after restarting RStudio itself to check that didn't fix it!) going to Tools -> Global Options and then Git/SVN and checking that "Enable Version Control" is ticked and there is a Git executable. If not, you'll need to reinstall git. If it is, I'd try unticking it, close the preferences and quit RStudio, and then opening RStudio and the Global Options again, ticking it, and restarting RStudio again and seeing if that helps?

**Reply by shayavh** [reply]
Hi @richardreeve,
I tried doing that and it didn't work unfortunately. I did install the latest update from apple, so that's probably it. There is a git executable and I followed all the steps but it is not appearing...

**Reply by richardreeve** [reply]
I would try installing xcode if you don't have it installed already. If you do, then try running it! It may well ask you to install something at that point. If that doesn't do anything, then at a terminal (in the macOS Terminal app or RStudio), try running:

```
xcode-select --install
```

Then try doing the unselect-restart-reselect-restart cycle above again?

**Comment by shayavh** [comment]
@richardreeve Thanks that worked! Got it back now! :)
